packages/@n8n/config/src/configs/database.ts

```diff
@@ -0,0 +1,159 @@
+import { Config, FromEnv, Nested } from '../decorators';
+
+@Config
+class LoggingConfig {
+	/** Typeorm logging enabled flag. @default false */
```

Comment:
I tend to think `@default` looks redundant and liable to drift. With IDE navigation we can find this out quickly now.